app/models/comment.rb

```diff
@@ -0,0 +1,9 @@
+class Comment < ApplicationRecord
+  belongs_to :author, class_name: 'User'
+  belongs_to :post, class_name: 'Post'
+  before_save :comments_counter
+
```

Comment:
- [ ] [OPTIONAL] Kindly use `after_save` method instead of `before_save` (in all models) as using `before_save` can result in the following issues:
   - Not showing the correct count as it would count the items before saving and thus showing the wrong result.
   - If the record has an issue and can't be saved for whatever database issue, the update counter method will be called as it gets called before saving and not after it saves the record.